std_msgs/Float64MultiArray q
std_msgs/Float64MultiArray v
std_msgs/Float64MultiArray tau
---
std_msgs/Float64MultiArray a